stdlib/3/subprocess.pyi

```diff
@@ -250,7 +250,12 @@ STDOUT = ...  # type: int
 if sys.version_info >= (3, 3):
     DEVNULL = ...  # type: int
     class SubprocessError(Exception): ...
-    class TimeoutExpired(SubprocessError): ...
+    class TimeoutExpired(SubprocessError):
+        cmd = ...  # type: List[str]
```

Comment:
Couldn't this also be `List[bytes]` or `str` (or even `bytes`)?

Reply:
Yeah, this could be all four of those things. Do we want to be opinionated in the typing here or permissive? I'll fall toward permissive and create a union of all four.